.github/workflows/eden_gcp.yml

```diff
@@ -0,0 +1,103 @@
+---
+name: EdenGCP
+on:  # yamllint disable-line rule:truthy
+  push:
+    branches: [master]
```

Comment:
So first things first @mydatascience  this overall Eden testing of master really has to go through the entire lifecycle of EVE and that has to wait for the publish workflow to complete. Hence I suggest you either trigger on `workflow_run` https://docs.github.com/en/actions/reference/events-that-trigger-workflows#workflow_run OR you can see if you can trigger on a release event `https://docs.github.com/en/actions/reference/events-that-trigger-workflows#release` since starting today every master build pushes artifacts to https://github.com/lf-edge/eve/releases/tag/0.0.0